### PR TITLE
Updates to use only npm for dependancies 

### PR DESCRIPTION
**Pull request by rafibomb**
Matching the updates done on the ZURB Stack.

closes https://github.com/zurb/foundation-sites/pull/10678

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,4 @@
 .DS_Store
-bower_components
```
Comment (ncoden) [inline .gitignore]: Should we really delete `bower_components` from `.gitignore` ?
We do not create the folder, but if one exists, it should still be ignored.

File: scss/app.scss [hunk 2]
```diff
@@ -52,4 +49,4 @@
 // @include foundation-prototype-classes;
 
 @include motion-ui-transitions;
-@include motion-ui-animations;
+@include motion-ui-animations;
```
Comment (ncoden) [inline scss/app.scss]: I don't think this is intended.

**Comment by ncoden** [comment]
@colin-marshall Could you take care of the requested changes ?

**Comment by colin-marshall** [comment]
@ncoden done.

**Comment by ncoden** [comment]
Note: installation guidelines must be updated in Foundation docs to remove `bower install`

**Comment by ncoden** [comment]
So I guess it can be merged ?

**Comment by DanielRuf** [comment]
This PR here includes changes that should not be in there.

File: scss/_settings.scss [hunk 3]
```diff
@@ -95,6 +95,7 @@ $global-menu-nested-margin: 1rem;
 $global-text-direction: ltr;
 $global-flexbox: true;
 $global-prototype-breakpoints: false;
+$global-button-cursor: pointer;
```
Comment (DanielRuf) [inline scss/_settings.scss]: Please reset the scss changes or do a rebase, they should not be in this PR imo.

**Comment by DanielRuf** [comment]
Sorry, it did not load the full changeset on mobile this morning.

**Comment by DanielRuf** [comment]
At least the title does not reflect the changes of the PR.

After this we can safely remove the `bower install` from the docs (or revert https://github.com/zurb/foundation-sites/pull/10678).

**Comment by DanielRuf** [comment]
Or are the scss changes safe to merge here? Personally I would split this into two PRs.

File: scss/_settings.scss (hunk 3 above)
Comment (ncoden) [reply scss/_settings.scss]: See 8b81373 : updated app.scss to match v6.4.3

Comment (DanielRuf) [reply scss/_settings.scss]: > See 8b81373 : updated app.scss to match v6.4.3

Ok, so this is still fine with the PR title? Just asking =)

Comment (ncoden) [reply scss/_settings.scss]: Foundation version on npm is `v6.4.4-rc1`. So we should also have all the changes between `v6.4.3` and `v6.4.4-rc1` here ;)

Comment (DanielRuf) [reply scss/_settings.scss]: Agreed.

Comment (ncoden) [reply scss/_settings.scss]: Done.

**Pull request merged**